### PR TITLE
Moving to subdir - Linux add edge node

### PR DESCRIPTION
**Pull request by guyhay**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
Let's copy - not move, I promise Jonathan we wouldn't do any more outright moves.

File: quickstarts/microsoft.hdinsight/101-hdinsight-linux-add-edge-node/azuredeploy.json [hunk 1]
```diff
@@ -10,31 +10,30 @@
     },
     "_artifactsLocation": {
       "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-hdinsight-linux-add-edge-node",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.hdinsight/101-hdinsight-linux-add-edge-node/azuredeploy.json:13]: change the defaultValue to ```[deployment().properties.templateLink.uri]```

File: quickstarts/microsoft.hdinsight/101-hdinsight-linux-add-edge-node/azuredeploy.json [hunk 2]
```diff
@@ -62,7 +60,7 @@
         "installScriptActions": [
           {
             "name": "[concat('emptynode','-' ,uniquestring(variables('applicationName')))]",
-            "uri": "[uri(parameters('_artifactsLocation'), concat(parameters('installScriptActionFolder'), '/', parameters('installScriptAction'), parameters('_artifactsLocationSasToken')))]",
+            "uri": "[concat(parameters('_artifactsLocation'), parameters('installScriptActionFolder'), '/', parameters('installScriptAction'), parameters('_artifactsLocationSasToken'))]",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.hdinsight/101-hdinsight-linux-add-edge-node/azuredeploy.json:63]: use: 
```[uri(parameters('_artifactsLocation'), concat(parameters('installScriptActionFolder'), '/', parameters('installScriptAction'), parameters('_artifactsLocationSasToken')))]```

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by guyhay** [comment]
fixed in another branch https://github.com/Azure/azure-quickstart-templates/pull/10747#event-4853943485

**Pull request closed without merging**